main.tf

```diff
@@ -62,14 +62,19 @@ resource "helm_release" "cloud_monitoring_agent" {
     value = local.cluster_name
   }
   set {
-    name  = "image.version"
+    name  = "global.imageRegistry"
     type  = "string"
-    value = var.image_tag_digest
+    value = var.image_registry
```

Comment:
I don't think this is correct? Does it work? I think the value here should be `icr.io/ext/sysdig`
Which is `[registry-url]/[namespace]`

Reply:
Ignore - I think I'm wrong